src/Libraries/RevitNodesUI/RevitTypes.cs

```diff
@@ -129,4 +129,39 @@ public class VerticalAlignment : CustomGenericEnumerationDropDown
         public VerticalAlignment() : base("Vertical Alignment", typeof(Autodesk.Revit.DB.VerticalAlignmentStyle)) { }
     }
 
+    [NodeName("Fill Patterns")]
```

Comment:
Yes, because this pull request is only meant to cherry-pick changes related to floor/roof, wallface, curtainsystem and familyinstance by geometry.